sig/datadog/appsec/actions_handler.rbs

```diff
@@ -0,0 +1,17 @@
+module Datadog
+  module AppSec
+    module ActionsHandler
+      def handle: (::Hash[::String, ::Hash[::String, ::String]] actions_hash) -> void
```

Comment:
Does it makes sense to define the `SecurityEngine::Runner::Result.action` type and use it here? If we break the contract, steep will complain, because now we can break it and never fine out

Reply:
Used the existing `Result::actions` and `Result.action` types [15afd41](https://github.com/DataDog/dd-trace-rb/pull/4300/commits/15afd41c8ca51c4d8bd4ca6b79dbbf0b65844487)